backend/app/api/mcp_routes.py

```
@@ -62,7 +67,7 @@ async def check_server_health(server_id: UUID, db: Session = Depends(get_db)):
         manager = MCPServerManager(db)
         health_status = await manager.health_check(server_id)
         return health_status
-    except Exception as e:
+    except Exception:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)